ingest/build-configs/manual-upload/README.md

```diff
@@ -9,19 +9,73 @@
 ## Run the workflow
 
 This workflow is expected to be run manually after downloading files from GISAID.
-Save the downloaded GISAID metadata and sequences as:
-- `ingest/build-configs/manual-upload/data/<YYYY-MM-DD-N>-metadata.xls`
-- `ingest/build-configs/manual-upload/data/<YYYY-MM-DD-N>-sequences.fasta`
+The GISAID files are expected to be saved as
+
+- `<YYYY-MM-DD-N>-metadata.xls`
+- `<YYYY-MM-DD-N>-sequences.fasta`
 
 `<YYYY-MM-DD>` is the date the files were downloaded from GISAID.
 `<N>` is the number of the download since GISAID limits the number of records per download.
 
 For example, if you had to split the data between two downloads on 2025-04-11,
 then save the files as
-- `ingest/build-configs/manual-upload/data/2025-04-11-1-metadata.xls`
-- `ingest/build-configs/manual-upload/data/2025-04-11-1-sequences.fasta`
-- `ingest/build-configs/manual-upload/data/2025-04-11-2-metadata.xls`
-- `ingest/build-configs/manual-upload/data/2025-04-11-2-sequences.fasta`
+- `2025-04-11-1-metadata.xls`
+- `2025-04-11-1-sequences.fasta`
+- `2025-04-11-2-metadata.xls`
+- `2025-04-11-2-sequences.fasta`
+
+The directory in which you save the GISAID files depends on which command you are
+using to run the workflow.
+
+### With `nextstrain run`
+
+When running with `nextstrain run`, you can save the GISAID files in any
+arbitrary analysis directory. However, you must also create a `config.yaml` within
+the analysis directory to specify the `gisaid_pairs` to upload.
+
+Continuing the example above, your analysis directory should look like
+```
+<analysis-dir>
+├── config.yaml
+└── data
+    ├── 2025-04-11-1-metadata.xls
+    ├── 2025-04-11-1-sequences.fasta
+    ├── 2025-04-11-2-metadata.xls
+    └── 2025-04-11-2-sequences.fasta
+```
+
+With the `config.yaml` specifying the `gisaid_pairs` you want to upload
+
+```yaml
+gisaid_pairs:
+ - 2025-04-11-1
+ - 2025-04-11-2
+```
+
+Make sure you have the latest seasonal flu pathogen setup.
+
+```
+nextstrain update seasonal-flu@master
+```
+
+Then run the workflow
+```
+nextstrain run \
```

Comment:
This use case looks great @joverlee521!

You may wish to note that you can run with snakemake as well, e.g. via

```
snakemake --snakefile ../path/to/seasonal-flu/ingest/build-configs/manual-upload --cores 1 ...
```

(or similar, I didn't test), which may be useful as it allows more fine-grained usage via additional snakemake arguments.

Reply:
Thanks James! I usually only include `nextstrain` commands and avoid `snakemake` commands because they would require users to install the workflow dependencies outside of the Nextstrain runtime.